### PR TITLE
Add Thapa Technical Youtube link

### LINKED ISSUE #667: I would like to add ThapaTechnical Youtube channel link
### Site URL

https://www.youtube.com/@ThapaTechnical

### Description of the above provided site

This channel is dedicated to the world of website development, offering a wide range of content related to technical aspects, tips and tricks, design principles, and programming. Whether you're a beginner looking to learn the basics or an experienced developer seeking advanced techniques, this channel aims to provide valuable insights and knowledge.

The channel covers various topics within website development, starting with the fundamentals of HTML, CSS, and JavaScript. It offers tutorials and explanations on how to create web pages, style them, and add interactivity using JavaScript. These videos can help you build a solid foundation in web development.

Programming videos on this channel cover different languages and frameworks commonly used in web development, such as Javascript, React, and more. These videos aim to help you understand the core concepts of each language and demonstrate how they can be used to build dynamic and interactive web applications.

Please assign me this issue under GSSOC 23 @rupali-codes 

### Category

Youtube

### Sub Category

Web Development

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Rahul-Bhati**
## Fixes Issue
Close #667 

## Changes proposed

Add Youtube link of Thapa Technical provide hindi content on web-development.

## Screenshots

Screenshot (1176)

### CODE REVIEW

File: database/youtube/web-development.json [hunk 1]
```diff
@@ -142,5 +142,13 @@
     "category": "youtube",
     "subcategory": "web-development",
     "language": "english"
+  },
+  {
+    "name": "Thapa Technical",
+    "description": "This channel is dedicated to the world of website development, offering a wide range of content related to technical aspects, tips and tricks, design principles, and programming.",
```
Comment (CBID2) [inline database/youtube/web-development.json:148]: [Proofreading]
```suggestion
    "description": "This channel is dedicated to the world of web development, offering a wide range of content related to technical aspects, tips and tricks, design principles, and programming.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Rahul-Bhati! I just made a proofreading suggestion in my inline comment. Check it out. 😃

**Review by Rahul-Bhati — COMMENTED** [review]
ok

**Review by CBID2 — APPROVED** [review]
Great job @Rahul-Bhati! :) LGTM! :)

**Review by rupali-codes — APPROVED** [review]
looks good

**Pull request merged**